airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml

```diff
@@ -2000,3 +2063,87 @@ schemas:
           - "null"
           - string
         format: date-time
+
+  contact_lists:
+    $schema: http://json-schema.org/draft-07/schema#
+    type:
+      - "null"
+      - object
+    additionalProperties: true
+    properties:
+      processingType:
+        type:
+          - "null"
+          - string
+      objectTypeId:
+        type:
+          - "null"
+          - string
+      updatedById:
+        type:
+          - "null"
+          - string
+      filtersUpdatedAt:
+        type:
+          - "null"
+          - string
+        format: date-time
+      listId:
+        type:
+          - "null"
+          - string
+      createdAt:
+        type:
+          - "null"
+          - integer
+      processingStatus:
+        type:
+          - "null"
+          - integer
+      deletedAt:
+        type:
+          - "null"
+          - string
+        format: date-time
+      listVersion:
+        type:
+          - "null"
+          - integer
+      name:
+        type:
+          - "null"
+          - string
+      additionalProperties:
+        type:
+          - "null"
+          - object
+        additionalProperties: true
+        properties:
```

Comment:
good catch on these additional properties being the default 5 when we don't included `additionalProperties` in the outbound request.

One request, can you add a comment here saying that we do not currently included the `additionalProperties` request parameter and Hubspot's API returns these 5 fields by default.

Reply:
added comment